src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1668,6 +1668,22 @@ private DiagnosticInfo GetBadEventUsageDiagnosticInfo(EventSymbol eventSymbol)
                 new CSDiagnosticInfo(ErrorCode.ERR_BadEventUsageNoField, leastOverridden);
         }
 
+        internal static bool IsReadingThroughBackingFieldInConstructor(BoundPropertyAccess propertyAccess, Symbol fromMember)
```

Comment:
I'm not sure the name is accurate.

Also regarding the "InConstructor" part. I added it since this will return `false` for a readonly old-style auto property if we're not in constructor, but will return true if we're in constructor.

Reply:
> Also regarding the "InConstructor" part. I added it since this will return ```false``` for a readonly old-style auto property if we're not in constructor, but will return true if we're in constructor.

I do not find it much helpful to keep "InConstructor" suffix. The method can return ```false``` due to many different reasons and that logic is incapsulated in it. There is really no need to reflect all that logic in the name of the method.